index.bs

```diff
@@ -461,9 +461,14 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     * the entry's [=attribution source/number of reports=] value is greater than 0.
 
     Note: This causes the user agent to favor triggering newer [=attribution sources=] over sources that have already been triggered.
-1. [=list/Remove=] all entries in |cache| where the entry's [=attribution source/expiry=] value is less than the current time.
+1. [=list/Remove=] all entries in |cache| where the entry's [=obtain an attribution source's expiry time|expiry time=] is less than the current time.
 1. If the [=list/size=] of |cache| is less than an implementation-defined limit, [=set/append=] |source| to |cache|.
 
+<h3 id="obtaining-attribution-source-expiry-time">Obtaining an attribution source's expiry time</h3>
+
+To <dfn>obtain an attribution source's expiry time</dfN> given an [=attribution source=] |source|:
```

Comment:
I would probably write this as

```suggestion
An [=attribution source=] |source|'s <dfn for="attribution source">expiry time</dfn> is the result of the following algorithm.
```

And since the following algorithm is a single line, it could even be:

```suggestion
An [=attribution source=] |source|'s <dfn for="attribution source">expiry time</dfn> is |source|'s [=attribution source/source time=] + |source|'s [=attribution source/expiry=].
```

I'm a little worried about "expiry" and "expiry time" being confusable, but it's probably clear enough.